caikit/core/model_management/local_model_trainer.py

```diff
@@ -98,11 +100,13 @@ def __init__(
 
             # Set up the worker and start it
             self._use_subprocess = use_subprocess
+            self._subprocess_start_method = subprocess_start_method
             if self._use_subprocess:
                 log.debug2("Running training %s as a SUBPROCESS", self.id)
                 self._worker = DestroyableProcess(
+                    start_method=self._subprocess_start_method,
                     target=self._train_and_save,
-                    return_result=True,
+                    return_result=False,
```

Comment:
Is this flag required in the implementation of `DestroyableProcess` at all?
Currently our only usage of `DestroyableProcess` is right here in the local trainer and I didn't think we had other immediate use cases for this.

Reply:
It's not _except_ if we want `DestroyableProcess` to match `DestroyableThread` which was the goal of adding it in the first place

Reply:
gotcha...
I'm not sure we need it to entirely match, and I wouldn't expect most things to be returnable from a subprocess anyway? /shrug

I'm fine either way, but I lean towards removing so that nobody stubs their toe in the future expecting this to return a loaded model, like we did today 😉

Reply:
Hm, totally valid. I'll compromise and make it default to False so that you need to opt in